apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/RdaClaimMessageMetaData.java

```diff
@@ -108,6 +108,12 @@ public class RdaClaimMessageMetaData {
   public static class RdaClaimMessageMetaDataBuilder {
     private Character claimType;
 
+    /**
+     * RdaClaimMessageMetaDataBuilder
```

Comment:
Please adjust the added javadocs to be compliant with checkstyle; method descriptions should be full sentences that describe the purpose of the method and end in a period. Something like "Creates a RdaClaimMessageMetaDataBuilder from a claim type value."

Reply:
ok. Will fix